cg/meta/demultiplex/delete_demultiplex_api.py

```diff
@@ -80,14 +80,16 @@ def active_samples_on_flow_cell(self) -> Optional[List[str]]:
     def _delete_sample_sheet_housekeeper(self) -> None:
         """Delete the presence of all sample sheets related to a flow cell in Housekeeper."""
         sample_sheet_files: Iterable[File] = self.housekeeper_api.files(
-            tags=[self.flow_cell_name, "samplesheet"]
+            tags=[self.flow_cell_name, SequencingFileTag.SAMPLE_SHEET.value]
```

Comment:
Probably out of scope for this PR but we maybe should look into if archived sample sheets should be considered.

Reply:
There are currently 1966 files tagged with archived_sample_sheet.

Reply:
fixed